src/main/kotlin/telegram/Handling.kt

```diff
@@ -666,81 +646,96 @@ object MafiaHandler {
-                showAdminListMenu(chatId, messageId, bot)
+                showAdminListMenu(chatId, long(0), bot, int(1))
             }
             parametrized(timeLimitOnCommand) {
-                val res = bot.sendMessage(ChatId.fromId(chatId), "Введите срок действия разрешения в днях:")
-                if (res.isSuccess) {
-                    val desc = res.get().messageId
-                    createAdminContext(desc, AdminState.HOST_TIME)
-                }
+                val msgId = sendMessage(
+                    bot,
+                    chatId,
+                    "Введите срок действия разрешения в днях:"
+                )
+                updateMessage(
+                    bot, chatId, msgId,
+                    inlineKeyboard {
+                        createAdminContext(msgId, AdminState.HOST_TIME)
+                    }
+                )
             }
             parametrized(timeLimitOffCommand) {
                 hostInfos.update(long(0)) { timeLimit = false }
-                showHostSettings(long(1), chatId, bot)
+                showHostSettings(chatId, long(1), bot, long(0))
             }
             parametrized(gameLimitOnCommand) {
-                val res = bot.sendMessage(ChatId.fromId(chatId), "Введите количество разрешенных игр:")
-                if (res.isSuccess) {
-                    val desc = res.get().messageId
-                    createAdminContext(desc, AdminState.HOST_GAMES)
-                }
+                val msgId = sendMessage(
+                    bot,
+                    chatId,
+                    "Введите количество разрешенных игр:"
+                )
+                updateMessage(
+                    bot, chatId, msgId,
+                    inlineKeyboard { createAdminContext(msgId, AdminState.HOST_GAMES) }
+                )
             }
             parametrized(gameLimitOffCommand) {
                 hostInfos.update(long(0)) { gameLimit = false }
-                showHostSettings(long(1), chatId, bot)
+                showHostSettings(chatId, long(1), bot, long(0))
             }
             parametrized(shareCommand) {
                 hostInfos.get(long(0))?.let {
                     hostInfos.update(long(0)) { canShare = !it.canShare }
-                    showHostSettings(long(1), chatId, bot)
+                    showHostSettings(chatId, long(1), bot, long(0))
                 }
             }
             parametrized(canReassignCommand) {
                 hostInfos.get(long(0))?.let {
                     hostInfos.update(long(0)) { canReassign = !it.canReassign }
-                    showHostSettings(long(1), chatId, bot)
+                    showHostSettings(chatId, long(1), bot, long(0))
                 }
             }
             parametrized(deleteHostCommand) {
                 hostInfos.delete(long(0))
-                showHostSettings(long(1), chatId, bot)
+                showHostSettings(chatId, long(1), bot, int(2))
```

Comment:
And now it's `int(2)` for some reason
It definitely looks like there's an error at least in one of those cases